src/AzureStack/KeyVaultAdmin/KeyVault.Admin/Microsoft.AzureStack.Management.KeyVault.Admin.csproj

```diff
@@ -0,0 +1,15 @@
+﻿
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.AzureStack.Management.KeyVault.Admin</PackageId>
+    <Description>Provides developers with libraries for the KeyVault admin under AzureStack.</Description>
+    <VersionPrefix>0.1.0-preview</VersionPrefix>
```

Comment:
We recommend starting preview versioning from `0.9.0-preview`

Reply:
We already have modules which were published starting at 0.1.0.